pkgs/racket-doc/scribblings/reference/contracts.scrbl

```diff
@@ -1665,18 +1665,12 @@ be blamed using the above contract:
 
 @defthing[predicate/c contract?]{
 
- Use this contract to indicate that some function is a
- predicate.
-
-  This contract also includes an optimization so that functions returning
-  @racket[#t] from @racket[struct-predicate-procedure?] are just returned directly, without
-  being wrapped. This contract is used by @racket[provide/contract]'s
-  @racket[struct] sub-form so that struct predicates end up not being wrapped.
-
- This contract is semantically equivalent to
- @racket[(-> any/c boolean?)] and
- @racket[(-> any/c boolean?)] also has the same optimization.
-
+ Equivalent to @racket[(-> any/c boolean?)]. This contract includes an optimization so that functions returning
```

Comment:
Personally, I would simply say:

> Equivalent to @racket[(-> any/c boolean?)]. Previously, this contract was necessary as it includes an additional optimization that was not included in `->`. Now however, @racket[->] performs the same
 optimization, so the contract should no longer be used. The contract is still provided for backwards compatibility.

I'll leave the decision whether to use `@deprecated` to you.

Reply:
Sounds good to me. I'll leave the `@deprecated` bit out since that seems unnecessarily strong.

Reply:
Is it "backwards compatibility" or "backward compatibility"?

Reply:
It's "backward" but I always get it wrong.